src/03-deals/deals.apib

```diff
@@ -77,6 +77,7 @@ Get a list of deals.
                 + estimated_value (Money)
                 + estimated_closing_date: `2017-05-09` (string)
                 + estimated_probability: `0.5` (number)
+                + summary: `Additional information` (string, nullable)
```

Comment:
PP: to me this goes hand in hand with the title, so I think it should go after `title` or at least closer to the top of the properties.

Reply:
Alright I can move it up.